fuse_examples/classification/knight/eval/eval.py

```diff
@@ -1,3 +1,21 @@
+"""
```

Comment:
I just skimmed this file in very high level, sorry :(

Reply:
NP, thanks